kernel/build/build_rpm

```diff
@@ -84,6 +84,19 @@ if [ "$THROWAWAY" = false ] ; then
     # push binaries to chacra
     find ~/rpmbuild/ | egrep '\.rpm$' | $VENV/chacractl binary ${chacra_flags} create ${chacra_endpoint}/$ARCH/
 
+    # write json file with build info
+    cat > $WORKSPACE/repo-extra.json << EOF
+{
+    "version":"$kernelrelease",
```

Comment:
@zmc Your wip-shaman-kernel branch uses `builder.version` to assign to `need_version[role]`.  That should become `builder.scm_version`.

Unless packaging.py *requires* `package_manager_version` - which I think it doesn't - we shouldn't need `package_manager_version` for the kernel task at all.